app/assets/stylesheets/layout_components/content-toggle.scss

```diff
@@ -34,3 +34,17 @@
     display: block;
   }
 }
+
+.edit_gift{
+  .associate-expandable{
+    display: none;
+  }
+
+  .join-checkout{
```

Comment:
would we be able to actually get rid of this class and in your other branch just change it to use .micetype where this class was being applied?